ui-admin/src/util/json/JsonEditor.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import CodeMirror from '@uiw/react-codemirror'
+import { json } from '@codemirror/lang-json'
+import { githubLight } from '@uiw/codemirror-theme-github'
```

Comment:
there are [other themes](https://uiwjs.github.io/react-codemirror/#/theme/home) if anyone has any preferences

Reply:
Any of the light ones look good imo- so what you've chosen works for me!